algorithm/scrypt/const.go

```
@@ -56,8 +56,16 @@ const (
 	ParallelismDefault = ParallelismMin
 )
 
+const (
+	AlgIdentifier = "scrypt"
```

golangci-lint (1.64.8)

[warning] exported: exported const AlgIdentifier should have comment (or a comment on this block) or be unexported

(revive)